package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "warden",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
You can use `$ npm version <type>` to bump this and write tags so everything stays in sync.

Not a big deal at all if you do it manually, just saves you some time.